### PR TITLE
Added Largest sum contiguous subarray in C

### PR DESCRIPTION
**Pull request by YashovardhanSiramdas**
**Changes:**
Given an array,program computes largest sum of contiguous subarray.

### CODE REVIEW

File: code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#include <stdio.h>
+
+int kadanesAlgorithm(int a[], int n)
```
Comment (arnavb) [inline code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c]: Function type on its own line.

File: code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{
+	int max_so_far = 0, max_ending_here = 0, i;
```
Comment (arnavb) [inline code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c]: Move i to its own line.

File: code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	
+	for (i = 0; i < n; i++) {
```
Comment (arnavb) [inline code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c]: Pre increment.

File: code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+	    if (max_so_far < max_ending_here)
+	        max_so_far = max_ending_here;
+	
+	    if (max_ending_here < 0)
+	        max_ending_here = 0;
+	}
+	return (max_so_far);
+}
+
+int main()
```
Comment (arnavb) [inline code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c]: Same as above.

File: code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+{
+	int n;
+	printf("Enter size of Array: ");
+	scanf("%d", &n);
+
+	int a[n], i;
```
Comment (arnavb) [inline code/dynamic_programming/src/largest_sum_contiguous_subarray/largest_sum_contiguous_subarray.c]: Same as above.

**Comment by YashovardhanSiramdas** [comment]
Made suggested Changes.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**